
---

# group configs
PlanGroupConfiguration[] group_configurations